demos/ripple.html

```diff
@@ -57,7 +57,14 @@
       }
 
       section {
-        margin: 48px;
+        align-items: center;
+        display: flex;
+
+      }
+
+      section > div {
+        margin: 48px 0;
+        width: 400px;
```

Comment:
should we add a media query here that sets it to `100%` width if the viewport is smaller than `400px`? I'm thinking of how this would appear on v. small screens